src/Components/customToastify.css

```diff
@@ -4,5 +4,5 @@
   --toastify-color-warning: var(--bs-warning);
   --toastify-color-error: var(--bs-danger);
 
-  --toastify-font-family: var(--jr-headline-font-family);
+  --toastify-font-family: var(--bs-body-font-family);
```

Comment:
Do we actually have to provide a `toastify-font-family` if we already use the body font? Is there some default from the library that we need to overwrite?

Reply:
Sure we can replace. I just handed over the right variable.

Reply:
I am not sure if I should replace it within the tastify component. I found several lines. I would leave it as it is for now.

Reply:
I just checked it. It's not working, since the default of `--toastify-font-family` is `sans-serif`. So we need to explicitly set the variable.